### ISSUE TITLE
短语音合成效果

### ISSUE THREAD

**Issue by shakingWaves** [xiaominfc/tacotron#1]
你好，针对短语音(小于六个字)合成效果比较差，不知道你有没有遇到这种情况？
还有就是，下面是跑了45600步的结果，感觉对齐的效果还不够理想。
![image](https://user-images.githubusercontent.com/3870673/64109062-1b27a800-cdb1-11e9-89c8-d52194bdc3bf.png)

**Comment by xiaominfc** [comment]
是的 确实是不好 已放弃改方案 改用https://github.com/CorentinJ/Real-Time-Voice-Cloning 了

**Comment by xiaominfc** [comment]
> 你好，针对短语音(小于六个字)合成效果比较差，不知道你有没有遇到这种情况？
> 还有就是，下面是跑了45600步的结果，感觉对齐的效果还不够理想。
> ![image](https://user-images.githubusercontent.com/3870673/64109062-1b27a800-cdb1-11e9-89c8-d52194bdc3bf.png)

是的 确实是不好 已放弃改方案 改用https://github.com/CorentinJ/Real-Time-Voice-Cloning 了

**Comment by xiaominfc** [comment]
> 你好，针对短语音(小于六个字)合成效果比较差，不知道你有没有遇到这种情况？
> 还有就是，下面是跑了45600步的结果，感觉对齐的效果还不够理想。
> ![image](https://user-images.githubusercontent.com/3870673/64109062-1b27a800-cdb1-11e9-89c8-d52194bdc3bf.png)

是的 确实是不好 已放弃了  正在尝试:  https://github.com/CorentinJ/Real-Time-Voice-Cloning 了
